owsi-core/owsi-core-components/owsi-core-component-jpa/src/main/java/fr/openwide/core/jpa/batch/processor/ThreadedProcessor.java

```diff
@@ -217,10 +218,10 @@ public void log(boolean force) {
 				if (StringUtils.hasText(loggerContext)) {
 					sb.append(loggerContext).append(" - ");
 				}
-				sb.append("Avancement {} / {} ({} ignorés, {} items / s. depuis début, {} items / s. depuis dernier log)");
+				sb.append("In progress: {} / {} ({} ignored, {} items/s since start, {} items/s since last log)");
 				
 				if (propertyService.get(MIGRATION_LOGGING_MEMORY)) {
```

Comment:
We could, but it would mean that *two* lines would appear in the logs (one for info, the other for debug), unless we add some mechanics to prevent the logging at info level if the debug level is enabled (which seems rather unconventionnal, but why not).